spec/support/factory_girl.rb

```diff
@@ -50,4 +53,11 @@
     longitude 67.4
     user
   end
+  factory :attachment do
+    sequence(:name) { |n| "Name#{n}" }
+    group
+    attachment { Rack::Test::UploadedFile.new(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.